actions/compatibility/entrypoint/main.go

```diff
@@ -0,0 +1,296 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/BurntSushi/toml"
+
+	msemver "github.com/Masterminds/semver"
+	"github.com/blang/semver"
```

Comment:
I am unclear on why you need both `Masterminds` and `blang` semver for this application. If you could point of the usage of both I would love to maybe try and whittle this down to one or the other.

Reply:
Much of this functionality was ported over directly from the concourse task, when I tried to touch it things starting falling apart. All of the old code used blang. Masterminds was pulled in with some new code. I'll convert all my changes to blang to match the rest.

Reply:
Done in https://github.com/paketo-buildpacks/github-config/pull/351/commits/69f4609180b6ef854efde20a4b994e57529d7e97